### DISCUSSION TITLE
Will you please update your documentation ?

### DISCUSSION THREAD

**Discussion by fzsf163** [codex-team/editor.js#2618] [General]
This is my first time building rte. I wanted to try editor js but the documentation is too ambiguous.  Most of the things needs to searched on the internet and go through every possible article and threads to find what I need. this documentation makes me feel like unwanted visitor.

**Comment by neSpecc** [comment]
It would be helpful if you could provide more details: an exact list of unclear sentences.

**Reply by fzsf163** [reply]
Only thing makes sense making it appear in the page. Let's say we all know it is a client side component. 
Say example :
```
const editor = new EditorJS();

editor.save().then((outputData) => {
  console.log('Article data: ', outputData)
}).catch((error) => {
  console.log('Saving failed: ', error)
});
```
wouldn't it make sense to give write an entire function to give this example. I am using remix. Where does it go? How should I save it to DB ? How do I later convert the data to HTML (I found there is an plugin for that). But the example helps. How about a total tutorial about how to really use it effectively  with all plugins and typescript. Because found few tutorials that do not really help. Like locally saving the image (e.g /public/img)(I understand that this is framework depened , but basics should be similar). Send the saved image url back to editor to display. Things that nowadays everyone needs. Instead of class how about function examples. How about set up a entire rich text editor with mostly used plugins (that includes images and videos etc ).

**Reply by neSpecc** [reply]
Thanks. It makes sense

**Reply by fzsf163** [reply]
Please help. I am losing my mind over this rich text editors. I really like the editor js style btw. That's why I am saying all this.

**Reply by tiagozip** [reply]
> How should I save it to DB ?

You should add your implementation, sending the JSON to your server

> Instead of class how about function examples

Classes are pretty easy to learn :D

> How do I later convert the data to HTML (I found there is an plugin for that)

Yeah you need to bring a plugin to do that.

> I am losing my mind over this rich text editors

I agree with you, normally text editors are pretty buggy and weird.

**Reply by fzsf163** [reply]
I will follow. Thank you
